browser/brave_vpn/win/service_details.cc

```diff
@@ -144,17 +144,9 @@ std::wstring GetBraveVpnWireguardTunnelServiceName() {
   NOTREACHED_NORETURN();
 }
 
-base::FilePath GetBraveVPNWireguardServiceInstallationPath(
```

Comment:
Nice to remove this - somewhere along the way, this wasn't called anymore and we had missed it

Reply:
Looks like it was removed in original PR https://github.com/brave/brave-core/pull/20754/files 😄👍